### DISCUSSION TITLE
How to organize family and friends?

### DISCUSSION THREAD

**Discussion by wsw70** [dani-garcia/vaultwarden#3759] [Q&A]
I am using Bitwarden/Vaultwarden for years and after 1000+ entries I decided to have a hard stare at how I share accounts. My target population is family and friends, without distinction.

So far I have created a "Family" organization with a few categories **related to people** and to be frank this is a bit chaotic.

My aim is to be able to share a specific account with specific people (one or more). What is the best organization/catoegory setup for this?

Is it to have one organization called "Everyone" and one category per person? And then invite everyone to "Everyone" and once they are invited assign them to a specific category (= their name)?

Would there be a way to gracefully transition from my current setup (an org "Family" with organizations "me + wife" and "me + wife + children")?

Or am I doing it wrong all the way?

**Comment by matlink** [comment]
There is not an unique solution that fits-them-all.
You should look at the "Groups" feature which is still in beta, but adds another way to group users for better flexibility.
Having a collection for each user you want to share a password with would be hard to manage, as you would probably share this password with 2 other persons and so on. The number of collections will grow rapidly, as you have `n!` combinations of persons you can share your passwords with.
For me, collections is a way of grouping shared passwords, not people. But again, this is my point of view, others may use it differently.

**Reply by wsw70** [reply]
> The number of collections will grow rapidly, as you have `n!` combinations of persons you can share your passwords with.

I am not sure I understand. If I have people `A` (assigned to collection `A`), `B` (assigned to collection `B`) etc. then for each record I would tick the appropriate collection(s) and these people would have access. I do not see where the factorial comes from?

**Reply by matlink** [reply]
Sure, you are right. It is not mandatory to have a collection for every combination possible.
Nonetheless, once you'll have a new user joining the organisation, you'll have to iterate over all your passwords to add them to the new collection. With groups or with semantic collection names, you will just have to add him to the groups/collections you need.
